schematron/v2.2.0/v2-2-0_L200_Audit.csv

```diff
@@ -78,4 +78,23 @@
 "","","","","","/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site/auc:Buildings/auc:Building/auc:Sections/auc:Section[auc:SectionType/text() = ""Whole building"" and auc:FootprintShape/text() = 'O-Shape']","false","","","count(auc:Sides/auc:Side) = 8","Incorrect number of auc:Side elements for footprint shape ""O-Shape"" (found <value-of select=""count(auc:Sides/auc:Side)""/>)","",""
 "","","","","","/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site/auc:Buildings/auc:Building/auc:Sections/auc:Section[auc:SectionType/text() = ""Whole building""]/auc:Sides/auc:Side","true","","","auc:WallIDs","","",""
```

Comment:
Change this check to (I believe) `auc:WallIDs/auc:WallID`, i.e. check that each side has atleast one WallID (this can remain as ERROR)

Reply:
With this error message: `Found an auc:Side with no linked auc:Wall`

Reply:
Resolved: https://github.com/BuildingSync/TestSuite/pull/37/commits/144bafc220b0ec40c591fc48ef08aa33cd7fc5ad